.gitignore

```diff
@@ -38,4 +38,5 @@ packages/*/coverage/
 # Generated files
 packages/cli/src/generated/
 .integration-tests/
+packages/vscode-ide-companion/dist
```

Comment:
@chrstnb / @skeshive this seems like a reasnoable change but something we should take separately.

@yuvrajangadsingh could you revert this bit